damus/ContentView.swift

```diff
@@ -473,6 +473,18 @@ struct ContentView: View {
         .onReceive(handle_notify(.new_mutes)) { notif in
             home.filter_muted()
         }
+        .onReceive(handle_notify(.logout)) { _ in
+            guard damus_state != nil else {
+                return
+            }
+
+            do {
+                try damus_state!.settings.delete_settings(damus_state!.pubkey)
+            } catch {
+                // Could not delete all settings for some reason. Continue with logout.
+                print("Unable to delete all user settings for \(damus_state!.pubkey). Continuing with logout.")
```

Comment:
This can technically happen when trying to delete the stored API keys for LibreTranslate and DeepL. Not sure how that could happen though. The vault deletion method throws so we have to catch it.